website/docs/d/image.html.markdown

```diff
@@ -34,6 +34,8 @@ resource "scaleway_server" "base" {
 
 * `name` - (Optional) Exact name of desired Image
 
+* `most_recent` - (Optional) Return most recent image if multiple exist
```

Comment:
Probably worth suffixing with "this cannot be used with the `name_prefix` field"?